### PR TITLE
Feat/about us

### PR DESCRIPTION
**Pull request by JudahRininger**
I added a carousel to the about page with cards displaying the pray team

## Description

I added a swiper slider with placeholder names, descriptions, and images to the about page and left the placeholder contact information on the top of the page.

## Reason

the purpose of this change is to provide a place to credit the developers

## How this has been tested

I tested scalability with the device toolbar in google chrome.

## Types of changes

- [x] New feature (adds functionality)
- [ ] Bug fix (fixes an issue)
- [ ] Breaking changes (causes defects to existing pieces)

## Screenshots
Screenshot 2024-03-21 at 3 01 05 PM

## Checklist:

- [ ] I have read and complied with the **CONTRIBUTING** document
- [ ] The changes I've made follow the existing code style and formatting requirements
- [ ] My changes require updates to the documentation, which I have modified accordingly

### CODE REVIEW

File: scriptured-prayer-components/src/assets/images/No_image_available.svg
Comment (asherlloyd) [inline scriptured-prayer-components/src/assets/images/No_image_available.svg:1]: Verify the source of this vector image

File: scriptured-prayer-components/src/components/TeamMemberCard.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import fallbackimage from "~/assets/images/No_image_available_white.svg";
+
+interface Props {
```
Comment (asherlloyd) [inline scriptured-prayer-components/src/components/TeamMemberCard.tsx:3]: Rename to `TeamMemberCardProps` for consistency as compared with other components.

**Review by asherlloyd — CHANGES_REQUESTED** [review]

File: scriptured-prayer-components/src/views/About.tsx
Comment (asherlloyd) [inline scriptured-prayer-components/src/views/About.tsx:1]: The background image is tiling; it should instead fully cover the background.

image

File: scriptured-prayer-components/src/views/About.tsx [hunk 2]
```diff
@@ -1,24 +1,35 @@
...
+            <h2 className="uppercase font-bold text-4xl">About us</h2>
+            <div className="my-8 text-xl">
+              <h3 className="font-bold uppercase">Phone Number</h3>
+              {/* change number */}
+              <div className="my-2">(123) 456-7890</div>
+            </div>
+            <div className="my-8 text-xl">
+              <h3 className="font-bold uppercase">Email Address</h3>
+              {/* change email */}
+              <div className="my-2">hello@scripturedprayer.com</div>
```
[the remark this answers is not in the document]
Comment (kenancasey) [reply scriptured-prayer-components/src/views/About.tsx:19]: I agree, especially about the phone number. My gut is that we should make a gmail address for the site (and forward it to @follindown?) as one point of contact.

File: scriptured-prayer-components/src/assets/images/No_image_available.svg
Comment (JudahRininger) [reply scriptured-prayer-components/src/assets/images/No_image_available.svg:1]: I got this image from Google icons, which is free to use and doesn't require any attribution to them.

File: scriptured-prayer-components/src/assets/images/No_image_available_white.svg
[the remark this answers is not in the document]
Comment (JudahRininger) [reply scriptured-prayer-components/src/assets/images/No_image_available_white.svg:1]: I got this image from Google icons, which is free to use and doesn't require any attribution to them.

File: scriptured-prayer-components/src/components/TeamMemberCarousel.tsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    {
+      name: "Asher Lloyd",
+      position: "Front-End Developer",
+      description:
+        "Designs and manages the user interface of Scriptured Prayer",
+    },
+    {
+      name: "Robbie Nichols",
+      position: "Freed Hardeman Alumn",
+      description: "Back-End Developer",
```
Comment (kenancasey) [inline scriptured-prayer-components/src/components/TeamMemberCarousel.tsx:37]: Spelling: `Backend` not `Back-End`

File: scriptured-prayer-components/src/views/About.tsx [hunk 4]
```diff
@@ -1,23 +1,41 @@
...
+          <div className="lg:flex px-6 py-8 lg:py-16 lg:space-x-12 lg:items-center max-w-screen-xl mx-auto grow">
+            <div className="max-w-[700px] mb-8">
+              <img src={dryGrass} />
+            </div>
+            <div>
+              <h2 className="uppercase font-bold text-4xl">About us</h2>
+              <div className="my-8 text-xl">
+                <h3 className="font-bold uppercase">Phone Number</h3>
+                {/* change number */}
+                <div className="my-2">(123) 456-7890</div>
```
Comment (kenancasey) [inline scriptured-prayer-components/src/views/About.tsx:20]: Are we even going to have a phone? I think we should just remove this.

**Review by kenancasey — CHANGES_REQUESTED** [review]
Please address these comments.

File: scriptured-prayer-components/src/views/About.tsx [hunk 5]
```diff
@@ -1,26 +1,35 @@
...
-            <div className="my-2">hello@scripturedprayer.com</div>
-          </div>
-          <Button>Our Privacy Policy</Button>
-        </div>
+        </Container>
+      </div>
+      <div className="bg-lichen">
+        <Container>
+          <h1 className="text-4xl text-center pt-5 pb-3 text-night">
+            Meet the Pray Team
```
Comment (LTsoccer) [inline scriptured-prayer-components/src/views/About.tsx:29]: Maybe just say Meet the Team.

File: scriptured-prayer-components/src/components/TeamMemberCarousel.tsx [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+    },
+    {
+      name: "Asher Lloyd",
+      position: "Front-End Developer",
+      description:
+        "Designs and manages the user interface of Scriptured Prayer",
+    },
+    {
+      name: "Robbie Nichols",
+      position: "Freed Hardeman Alumn",
```
Comment (LTsoccer) [inline scriptured-prayer-components/src/components/TeamMemberCarousel.tsx:36]: I would swap position and description here to match the others more

**Review by LTsoccer — APPROVED** [review]
Works locally. Code looks decent. Suggested changes to some wording issues.

**Review by LouviersJ — APPROVED** [review]
I agree with Lauren just saying "Meet the team" would be more cogent. This is some good code!

**Comment by LouviersJ** [comment]
Pagination kinda stinks but that should be resolved with my pull request

**Pull request closed without merging**